lti_consumer/models.py

```diff
@@ -316,10 +354,33 @@ def _get_lti_1p3_consumer(self):
                 consumer.enable_nrps(get_lti_nrps_context_membership_url(self.id))
 
             return consumer
+        elif self.config_store == self.CONFIG_ON_DB:
+            # TODO: I couldn't quite tell whether the Proctoring Services Specification relies on LTI Advantage or
+            #       just LTI 1.3, but I think, according to this document, it's a proposed LTI Advantage service,
+            #       so I'm using the LtiAdvantageConsumer.
+            #       https://www.imsglobal.org/activity/learning-tools-interoperability
+            consumer = LtiAdvantageConsumer(
+                iss=get_lms_base(),
+                lti_oidc_url=self.lti_1p3_oidc_url,
+                lti_launch_url=self.lti_1p3_launch_url,
+                client_id=self.lti_1p3_client_id,
+                # Deployment ID hardcoded to 1 since
+                # we're not using multi-tenancy.
+                deployment_id="1",
+                # Model Private RSA Key
+                rsa_key=self.lti_1p3_private_key,
+                rsa_key_id=self.lti_1p3_private_key_id,
+                # Model LTI 1.3 Tool Key/Keyset URL
+                tool_key=self.lti_1p3_tool_public_key,
+                tool_keyset_url=self.lti_1p3_tool_keyset_url,
+            )
+            # I am omitting the various other LTI Advantage services that can be set up, as they're not relevant
```

Comment:
Can you raise an exception if this is anything other than a proctoring configuration?

Reply:
How do you recommend doing that? Right now, that kind of information is stored on the xBlock, and there isn't an analogous place when storing configuration in the database. Is the right approach to add a `proctoring_enabled` field to the model?